llmfoundry/utils/config_utils.py

```diff
@@ -97,3 +97,11 @@ def log_config(cfg: DictConfig):
             raise e
         if wandb.run:
             wandb.config.update(om.to_container(cfg, resolve=True))
+
+    if 'mlflow' in cfg.get('loggers', {}):
+        try:
+            import mlflow
+        except ImportError as e:
+            raise e
+        if mlflow.active_run():
```

Comment:
I think [this](https://mlflow.org/docs/latest/tracking.html#logging-functions) is what I want?

Reply:
actually, we tested it last week. we're good!